src/openforms/submissions/tests/factories.py

```diff
@@ -256,29 +255,32 @@ class Meta:
     @classmethod
     def create(
         cls,
+        form_key: str = None,
```

Comment:
```suggestion
        form_key: Optional[str] = None,
```
or use the empty string as default